gravitee-am-gateway/gravitee-am-gateway-handler/gravitee-am-gateway-handler-oidc/src/test/java/io/gravitee/am/gateway/handler/oauth2/service/token/TokenServiceTest.java

```diff
@@ -121,6 +124,38 @@ public void shouldCreate() {
         verify(refreshTokenRepository, never()).delete(anyString());
     }
 
+    @Test
```

Comment:
should it be a test with an empty list too?

Reply:
Made a change into the service code to force the first entry to be the client_id what ever is the result of the ExpressionLanguage.